### PR TITLE
Added clarification text to the eth1 endpoint option.

### PR DESCRIPTION
**Pull request by bgravenorst**
## Pull Request Description
Added clarification text to the eth1 endpoint option.

### CODE REVIEW

File: docs/Reference/CLI/CLI-Syntax.md [hunk 1]
```diff
@@ -314,11 +314,11 @@ receiving warnings that the ETH1 node is unavailable.
...
 
-The JSON-RPC URL of Ethereum 1.0 node.
+The JSON-RPC URL of Ethereum 1.0 node. This option must be specified if running a validator.
 
-If not specified, then provide an initial state using the [`--initial-state`](#initial-state) option, or
-start Teku from an existing database using [`--data-path`](#data-path), which provides the initial
-state to work from.
+If not specified (that is, you're running a beacon chain client only), then provide an initial state
+using the [`--initial-state`](#initial-state) option, or start Teku from an existing database using
+[`--data-path`](#data-path), which provides the initial state to work from.
```
Comment (ajsutton) [inline docs/Reference/CLI/CLI-Syntax.md:321]: If you're running the already started public networks like MainNet or Pyrmont then you don't need to provide an initial state either - Teku already has it built in.

Comment (bgravenorst) [reply docs/Reference/CLI/CLI-Syntax.md]: Implemented suggestion.

**Review by ajsutton — APPROVED** [review]
LGTM.

File: docs/Reference/CLI/CLI-Syntax.md (hunk 1 above)
Comment (bgravenorst) [reply docs/Reference/CLI/CLI-Syntax.md]: Thanks @ajsutton, could you please approve this if you agree with the change?

Comment (bgravenorst) [reply docs/Reference/CLI/CLI-Syntax.md]: Sorry, beat me to it :)

**Pull request merged**